tox.ini

```
@@ -1,8 +1,8 @@
 [tox]
 skipsdist=True
 envlist=lint,
-        py36-dj{111,20}-wag23,
-        py{36,38}-dj{20,22}-wag28,
+        py{36}-dj{111,22}-wag23,
+        py{36,38}-dj22-wag29,
```

These can still be factors

```suggestion
    py{36}-dj{111,22}-wag{23},
    py{36,38}-dj{22}-wag{29}
```

---

Done